components/f4jumble/README.md

```diff
@@ -0,0 +1,20 @@
+# f4jumble
+
+TBD
```

Comment:
An implementation of the F4Jumble algorithm.

F4Jumble is used in Zcash Unified Addresses to prevent certain kinds of
address malleation attacks, and is specified by [ZIP 316](https://zips.z.cash/zip316).